rag/vectordb/postgres_node_storage.py

```diff
@@ -60,9 +60,11 @@ def store_nodes(self, nodes: List[str], access_level: int):
         dimension=1536
     )
 
-    processor = TXTProcessor(DOCUMENT_PATH)
+    processor = PDFProcessor(DOCUMENT_PATH)
     nodes = processor.process()
-
+    
+    print(nodes)
```

Comment:
Do you need this print?

Reply:
Considering that it's just a test code not to be ran in the production code, I think it's fine.

Reply:
Yes, it's just a test code